fiftyone/operators/operator.py

```
@@ -6,11 +6,33 @@
+def _normalize_risk_level(risk_level: str | RiskLevel | None) -> RiskLevel:
+    if risk_level is None:
+        return RiskLevel.DANGEROUS
+
+    if isinstance(risk_level, str):
+        try:
+            return RiskLevel[risk_level.upper()]
+        except KeyError:
+            raise ValueError(
+                "Invalid risk level '%s'. Valid values are: %s"
+                % (risk_level, [r.value for r in RiskLevel])
+            )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)